package.json

```diff
@@ -16,7 +16,7 @@
       "transform-class-properties"
     ]
   },
-  "version": "5.0.3",
+  "version": "1.0.0",
```

Comment:
let's not change the version
npm publish pipeline would take care of this
we'll need to include this package in the pipeline

Reply:
updated